.github/workflows/generate_new_client_hermetic_build.yaml

```diff
@@ -0,0 +1,157 @@
+name: Generate new GAPIC client library (Hermetic Build)
+on:
+  workflow_dispatch:
+      # some inputs are ommited due to limit of 10 input arguments
+      inputs:
+        api_shortname:
+          required: true
+          type: string
+          description: "`api_shortname`: Name for the new directory name and (default) artifact name"
+        name_pretty:
+          required: true
+          type: string
+          description: "`name_pretty`: The human-friendly name that appears in README.md"
+        api_description:
+          required: true
+          description: "`api_description`: Description that appears in README.md"
+        proto_path:
+          required: true
+          type: string
+          description: |
+            `proto_path`: Path to proto file from the root of the googleapis repository to the
+            directory that contains the proto files (without the version).
```

Comment:
Why not just use the versioned proto_path? I think it fits better since we can just add it as a GAPIC directly.

Reply:
This is to match the existing `new-client.py` script. The old one accepts root-level `proto_path`s, so a PR with all the released versions get created. See [this example PR](https://github.com/googleapis/google-cloud-java/pull/10485)

Reply:
I was initially forcing the user to provide a versioned path until I saw the behavior of the old script

Reply:
I don't think we have to follow the same behavior, its a tool that is used only by our own team, if the new behavior means less code and no additional toil, then we should go for it.

Reply:
Agreed. As discussed, I modified the script to support versioned proto_paths only.